JS/fluentRevealNavbar.uc.js

```diff
@@ -38,15 +52,24 @@
 
     // get all the toolbar buttons in the navbar, in iterable form
     get toolbarButtons() {
-      let buttons = Array.from(
-        gNavToolbox.querySelectorAll(".toolbarbutton-1")
-      );
-      if (this._options.includeBookmarks) {
-        buttons = buttons.concat(
-          Array.from(this.placesToolbarItems.querySelectorAll(".bookmark-item"))
+      if (!this._toolbarButtons || this._options.cacheButtons == false) {
+        this._toolbarButtons = Array.from(
+          gNavToolbox.querySelectorAll(".toolbarbutton-1")
         );
+        if (this._options.includeUrlBar) {
+          this._toolbarButtons.push(gNavToolbox.querySelector("#urlbar-background"));
+        }
+        if (this._options.includeBookmarks) {
+          this._toolbarButtons = this._toolbarButtons.concat(
+            Array.from(
+              this.personalToolbar.querySelectorAll(
+                ".toolbarbutton-1, .bookmark-item"
+              )
+            )
+          );
+        }
       }
-      return buttons;
+      return this._toolbarButtons;
     }
 
     get placesToolbarItems() {
```

Comment:
We need this deleted. personalToolbar is supposed to replace this

Reply:
By that I mean `placesToolbarItems`. Github's display of my comments is really weird